Userland/Libraries/LibWasm/AbstractMachine/Operators.h

```diff
@@ -216,7 +216,7 @@ struct VectorShiftRight {
     auto operator()(u128 lhs, i32 rhs) const
     {
         auto shift_value = rhs % (sizeof(lhs) * 8 / VectorSize);
-        return bit_cast<u128>(bit_cast<Native128ByteVectorOf<NativeIntegralType<128 / VectorSize>, SetSign>>(lhs) >> shift_value);
+        return bit_cast<u128>(bit_cast<NativeVectorType<128 / VectorSize, VectorSize, SetSign>>(lhs) >> shift_value);
```

Comment:
I'd like some explanation on the commit description (similar to the next one), otherwise LGTM.